CHANGES.rst

```diff
@@ -230,6 +230,7 @@ Bug fixes
 - ``astropy.io.ascii``
 
 - ``astropy.io.fits``
+  - Removed raising of ``AssertionError`` that could occur after closing or deleting compressed image data. [#4690]
```

Comment:
The line should be wrapped a little more tightly too--don't make it longer than other lines in the file. Just makes it easier to read.